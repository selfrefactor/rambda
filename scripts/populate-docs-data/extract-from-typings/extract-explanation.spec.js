import { extractExplanation } from './extract-explanation.js'

test('happy', () => {
  expect(extractExplanation()).toMatchSnapshot()
})
