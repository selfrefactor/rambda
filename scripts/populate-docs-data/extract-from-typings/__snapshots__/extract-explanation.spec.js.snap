// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`happy 1`] = `
Object {
  "add": "It adds \`a\` and \`b\`. It doesn't work with strings, as the inputs are parsed to numbers before calculation.

This is test",
  "adjust": "It replaces \`index\` in array \`list\` with the result of \`replaceFn(arr[i])\`.",
}
`;
